mesh/exceptions/InvalidJsonFormat.py

```diff
@@ -0,0 +1,8 @@
+from .BaseCustomException import BaseCustomException
+
+class InvalidJsonFormat(BaseCustomException):
```

Comment:
I love the exception handling organization with the base and child specific class